docs/markdown/glossary.md

```diff
@@ -0,0 +1,260 @@
+# Glossary
+
+This glossary provides definitions for key terms in the Interledger ecosystem.
+
+## Account
+   
+An accounting relationship between two Interledger participants. In Interledger, two participants establish an account with one another to track the current obligations they hold with one another.                                                                         
+
+## Address
+
+See instead: [Interledger Address](#interledger-address)
+
+## Application Layer
+
+The set of protocols built upon the Transport Layer that communicate payment details between senders and receivers. 
+
+Application Layer protocols specify all of the data and methods required to set up a payment for a specific category of use cases or applications. For example, the Simple Payment Setup Protocol uses WebFinger and HTTPS for account detail lookup.                                                                       
+
+## Arbitrage
+
+The practice of doing cyclic transactions which start and end on the same ledger, or groups of simultaneous transactions which add up to one cyclic transaction. The goal is to achieve a destination amount which is higher than the source amount by taking advantage of fluctuations in exchange rates.                                                                       
+
+## Asset
+
+Something you can own. Can be a physical and unique item, a quantity of a physical substance, a specific physical collection of coins and banknotes, a non-physical quantity of currency, a non-physical claim to a resource, a reputation for keeping one's word, someone else's intention to obey the terms of a contract, etc. Some types of assets can be more easily traded on a ledger than others. Also, for some asset pairs it is easier to determine a fair exchange rate than for others.                                                                       
```

Comment:
> It might be useful to bring in the words "fungible" and "non-fungible"

I agree with @emschwartz. While assets in general can be more broadly defined, we should probably keep our terminology limited to fungible assets. Maybe this...?
 
```suggestion
Something you can own that is fungible. Can be a physical item; a quantity of a physical substance; a specific physical collection of coins or banknotes; a non-physical quantity of currency; a non-physical claim to a resource; a reputation for keeping one's word; someone else's intention to obey the terms of a contract; etc. Some types of assets can be more easily traded on a ledger than others. For some asset pairs it is easier to determine a fair exchange rate than for others.                                                                       
```